### ISSUE TITLE
doesn't sync settings between computers

### ISSUE THREAD

**Issue by aspiers** [stsquad/emacs_chrome#160]
The option settings don't get stored in the user profile and synced across devices.  It also means that if a local Chrome user profile is wiped, the settings don't get restored when the user signs in again.

**Comment by stsquad** [comment]
@aspiers I assume this is a Google service that extensions can take advantage of? Otherwise this is out of scope for the extension as we'd need infrastructure to support that.

**Comment by aspiers** [comment]
Yes, this is a mainstream API which countless other extensions have taken
advantage of since it was introduced in Chrome 29:

https://developer.chrome.com/apps/storage
